app/controllers/members_controller.rb

```diff
@@ -0,0 +1,76 @@
+class MembersController < ApplicationController
+	before_action :require_login
+
+	def new
+		@member = Member.new
+	end
+
+	def create
+		p params
+		@member = Member.new(member_params)
+		@member.family_id = current_user.id
+		@member.save
+		@member.assign_color
+		@member.save!
```

Comment:
Why save then color then save! - this will look strange to readers. Also remove the debugging p statements in master.